clientconn.go

```diff
@@ -75,8 +75,6 @@ var (
 	// errConnClosing indicates that the connection is closing.
 	errConnClosing = errors.New("grpc: the connection is closing")
 	errNoAddr      = errors.New("grpc: there is no address available to dial")
-	// minimum time to give a connection to complete
-	minConnectTimeout = 20 * time.Second
```

Comment:
I saw line#586 (before this change) sets it to copts and then copts is passed transport.NewClientTransport.